common/src/main/scala/org/apache/comet/CometConf.scala

```diff
@@ -276,6 +276,14 @@ object CometConf {
       .booleanConf
       .createWithDefault(false)
 
+  val COMET_EXPLAIN_FALLBACK_ENABLED: ConfigEntry[Boolean] =
```

Comment:
Do we really need to make this configurable?

Reply:
I think a config should be good. Sometimes it might be verbose.